src/include/platform/ConfigurationManager.h

```diff
@@ -71,7 +71,9 @@ class ConfigurationManager
         kMaxProductLabelLength          = 64,
         kMaxSerialNumberLength          = 32,
         kMaxUniqueIDLength              = 32,
-
+#if CHIP_ENABLE_ROTATING_DEVICE_ID
+        kRotatingDeviceIDUniqueIDLength = 16,
```

Comment:
Why 16?  And is that a fixed length, or a max length?

Reply:
Actually that is min length. Spec says it should be 128-bit or longer, but it doesn't precisely determine the max length, so it is hard to allocate a buffer for that. Do you think I should create a spec issue for that?

Reply:
I think the intent is that any specific implementation would use whatever it wants.

In our case, maybe we should have a CHIP_CONFIG variable for it?

Reply:
Ok, I created a config that defines used unique ID length and additionally there is enum describing the minimum length of 16 bytes.